api/cms.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"errors"
+	"github.com/spf13/viper"
+	"gopkg.in/gin-gonic/gin.v1"
+	"os"
+)
+
+func CMSPerson(c *gin.Context) {
+	id := c.Query("id")
+	fbLang := c.DefaultQuery("fbLang", "en")
```

Comment:
language and it's fallback are in consts.go

Reply:
done